Gemfile

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+gem "webrick"
+gem 'public_suffix', '3.0'
+gem 'addressable', '2.8.5'
+gem 'rexml', '3.2.6'
+gem 'kramdown', '2.3.0'
+gem "kramdown-parser-gfm"
+gem 'rouge', '3.19.0'
+gem 'jekyll', '3.9.0'
+gem 'jekyll-archives', '2.1.1'
+gem 'jekyll-sass-converter'
+gem 'jekyll-feed', '0.13.0'
+gem 'jekyll-seo-tag', '2.6.1'
+gem 'jekyll-sitemap', '1.4.0'
+gem 'jekyll-paginate', '1.1.0'
+gem 'liquify', '0.2.7'
+gem 'github-pages', '207'
+
+# gem 'jekyll', '3.8.4'
+# gem "kramdown", ">= 2.3.1"
+# group :jekyll_plugins do
+#   gem 'jekyll-archives', '2.1.1'
+#   gem 'jekyll-feed', '0.11.0'
+#   gem 'jekyll-paginate', '1.1.0'
+#   gem 'jekyll-seo-tag', '2.5.0'
+#   gem 'jekyll-sitemap', '1.2.0'
+# end
```

Comment:
I think it's best to remove this; if we need to consult the old versions we can just look at the file history in git

Reply:
thanks, sorted